src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```
@@ -335,6 +334,17 @@ FINANCIAL_REPORTS:
     ROOT_PATH: sandbox
     STORAGE_TYPE: localfs
 FOOTER_ORGANIZATION_IMAGE: /images/logo.png
+FORUM_SEARCH_BACKEND: "forum.search.es.ElasticsearchBackend"
+
+FORUM_ELASTIC_SEARCH_CONFIG:
+  - host: "{{ keyOrDefault \"elasticsearch/host\" \"elasticsearch.service.consul\" }}"
+    port: "443"
+    use_ssl: true
+
+
+FORUM_MONGODB_CLIENT_PARAMETERS: mongo_params
```

For using YAML references you would do `*mongo_params`, but it also needs different parameters than the edxapp DB. This should go in the top block where the secrets get managed and look like:
```suggestion
{{ with secret "secret-mitxonline/mongodb-forum" }}
FORUM_MONGODB_CLIENT_PARAMETERS:
  authSource: admin
  host: {{ keyOrDefault "mongodb/host" "mongodb-master.service.consul" }}
  port: 27017
  db: forum
  replicaSet: {{ keyOrDefault "mongodb/replica-set" "rs0" }}
  user: {{ .Data.username }}
  password: {{ .Data.password }}
  ssl: {{ keyOrDefault "mongodb/use-ssl" "false" }}
{{ end }}
```